### PR TITLE
Added 'rotationIsDir' to ParticleSystem

### PR DESCRIPTION
**Pull request by elipsitz**
It allows people to set the particles in a system to have the same rotation as their direction. Useful for certain particle effects.

### CODE REVIEW

**Comment by dumganhar** [comment]
Thanks, could you make a test demo to ParticleTest?

**Comment by elipsitz** [comment]
Added the test for rotationIsDir

**Comment by dumganhar** [comment]
Thank you, I will test it ASAP.

File: cocos2dx/particle_nodes/CCParticleSystem.h [hunk 1]
```diff
@@ -196,6 +196,8 @@ class CC_DLL CCParticleSystem : public CCNode, public CCTextureProtocol
         float radialAccel;
         /** radial acceleration variance of each particle. Only available in 'Gravity' mode. */
         float radialAccelVar;
+        /** set the rotation of each particle to its direction Only available in 'Gravity' mode. */
+        bool rotationIsDir
```
Comment (dumganhar) [inline cocos2dx/particle_nodes/CCParticleSystem.h]: Missing semi-colon at the end of this line.

File: samples/Cpp/TestCpp/Resources/Particles/lines.plist [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    <key>startParticleSize</key>
+    <real>60</real>
+    <key>startParticleSizeVariance</key>
+    <real>10</real>
+    <key>tangentialAccelVariance</key>
+    <real>0</real>
+    <key>tangentialAcceleration</key>
+    <real>15</real>
+    <key>textureFileName</key>
+    <string>img/line.png</string>
```
Comment (dumganhar) [inline samples/Cpp/TestCpp/Resources/Particles/lines.plist]: There isn't a file named `img/line.png`.

**Comment by elipsitz** [comment]
Wow, that's an embarrassing mistake. Fixed.

**Comment by dumganhar** [comment]
Hi @ThePickleMan 
Is `rotationIsDir` supported by Particle Tools like `Particle Designer`?

**Comment by elipsitz** [comment]
Not that I'm aware of.

**Comment by elipsitz** [comment]
In my tests, I don't think there was a problem with that in the plists. I
suppose it is up to the game programmer to do it either manually or just
make the particle via code rather than a designer/plist.

**Comment by dumganhar** [comment]
Make sense. Thanks. :)

**Comment by dumganhar** [comment]
Merged. Thanks. :)

**Pull request merged**